src/ghc-specific/Foreign/JQuery.hs

```diff
@@ -24,6 +24,9 @@ setValueForId _ _ = return ()
 setHtmlForId :: String -> String -> IO ()
 setHtmlForId _ _ = return ()
 
+activateScriptTags :: String -> IO ()
+activateScriptTags _ = return ()
+
 show :: String -> IO ()
 show _ = return ()
```

Comment:
Because as we have our application right now, transient requires to have the same code on the client and the server. If the code is GHCJS-specific, it needs to be replaced with these empty `return` s